solr/core/src/test/org/apache/solr/security/JWTAuthPluginIntegrationTest.java

```diff
@@ -127,13 +135,40 @@ public void tearDown() throws Exception {
     super.tearDown();
   }
 
-  @Test(expected = IOException.class)
-  public void infoRequestWithoutToken() throws Exception {
-    get(baseUrl + "/admin/info/system", null);
+  @Test
+  public void extractCertificateFromPem() throws IOException {
```

Comment:
Can we have a test where the pen file or directly supplied string is improperly formatted? I'd like to see what kind of error message users can expect in that case (and maybe we should document that, because I suspect it will be cryptic otherwise)

Reply:
I added two tests to JWTAuthPluginTest, verifying that init() fails. No need for integration test of all these? 

If you load a `security.json` to ZK with bogous `trustedCertsFile` or `trustedCerts`, the JWT plugin will throw and exception and auth will not be enabled. Here are log messages you can expect to see prior to exception:

> Reading trustedCerts from file {}

or

> Reading trustedCerts PEM from configuration string

In the exception stacktrace itself  you'll see either

> Wrong type of certificates. Must be DER or PEM format

or

> Failed loading certificate(s) from input stream